### DISCUSSION TITLE
Latest stable - profiles

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15105] [How to]
Gents, my exec_time in php is 5200, yet still cannot save some module access in settings. Were else it can be controlled?

**Comment by mariuszkrzaczkowski** [comment]
attach server configuration report (Conf Report)
check parameters: output_buffering , post_max_size , max_input_vars  , memory_limit,

**Reply by amineb001** [reply]
![ConfReport(1)](https://github.com/YetiForceCompany/YetiForceCRM/assets/134292800/652b8f03-bb9b-4443-b573-be926ffb3a56)
I have a same problem i checked these parameters and nothing seems incorrect.
Could you check please and advise me ? @mariuszkrzaczkowski

**Reply by mariuszkrzaczkowski** [reply]
there are a lot of bugs, even a problem with file permissions
If you don't fix it, the system won't work properly

**Reply by amineb001** [reply]
@mariuszkrzaczkowski  Could you please specify exactly what should be fixed and give me some useful links?
I've been struggling with this problem for a while now, doing various upgrades hasn't solved the problem either.
Thank you very much.

**Comment by mariuszkrzaczkowski** [comment]
everything that is red needs to be corrected,
I don't know your environment so I can't give you good advice.
it's best to go to the server admin
https://doc.yetiforce.com/introduction/requirements/
https://doc.yetiforce.com/developer-guides/environments/debian-10
